CONTRIBUTING.md

```diff
@@ -1,5 +1,39 @@
 # Contributing
 
+## Templates update
```

Comment:
@r00ta @danielezonca can you also have a look at this section? It will be good to have similar contribution guide/pattern across the two template repositories.